bufr2geojson/__init__.py

```diff
@@ -144,20 +134,38 @@ def __init__(self):
             "22": {}  # some sst sensors in class 22
         }
 
+        self.code_table = {}
+        csv_code_table = f"{TABLES}{os.sep}BUFRCREX_CodeFlag_en.txt"
+
+        with open(csv_code_table) as csvfile:
+            reader = csv.DictReader(csvfile)
+
+            # keep only numeric rows, strip out non numeric rows,
+            # these typically give a range
+            for row in reader:
+                try:
+                    row['CodeFigure'] = int(row['CodeFigure'])
+                    self.code_table[row['FXY']] = row
```

Comment:
I think this breaks the code table. There will be multiple rows with the same FXY value, we are only keeping the last.